rfcs/popover-component.md

```diff
@@ -2,7 +2,31 @@
 
 ## Usage
 
-```js
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {StatefulPopover as Popover} from 'baseui/popover';
+
+export default () => {
+  const popoverContent = <div>Hello world</div>;
+  return (
+    <React.Fragment>
+      <Popover
+        placement="topLeft"
+        content={popoverContent}
+      >
+        <span>Hover me!</span>
```

Comment:
I think maybe my original example was bad here, but by default Popovers are triggered by click, so maybe this should be "Click me!". Maybe in advanced usage we can add `triggerType="hover"`.